pegasus_wrapper/nuke_checkpoints.py

```diff
@@ -0,0 +1,13 @@
+import sys
```

Comment:
This file looks like it needs some documentation about how to use inside of a workflow.